portal/templates/reporting_dashboard.html

```diff
@@ -6,6 +6,9 @@ <h3 class="tnth-headline">{{ _("Reporting Dashboard") }}</h3>
 
     <h2 class="tnth-headline">{{ _("Usage Statistics") }}</h2>
     <div>
+        <p>{{ _("Registrations are collected from the User.registration timestamps of all Users without the Test Role") }}</p>
```

Comment:
@mcjustin did you file this one already?  Not critical, but a pretty easy fix and the current info would be misleading.